pkg/entrypoint/entrypointer.go

```diff
@@ -189,7 +192,7 @@ func (e Entrypointer) Go() error {
 
 	// strings.Split(..) with an empty string returns an array that contains one element, an empty string.
 	// This creates an error when trying to open the result folder as a file.
-	if len(e.Results) >= 1 && e.Results[0] != "" {
+	if e.ResultExtractionMethod == config.ResultExtractionMethodTerminationMessage && len(e.Results) >= 1 && e.Results[0] != "" {
```

Comment:
This probably isn't the right place for this check - this is also going to skip other things like spire signing, which we'd want to preserve.

This should probably be a switch around `termination.WriteMessage` below.

Reply:
Good point. I will do that!